cmd/version.go

```diff
@@ -0,0 +1,33 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/operator-framework/combo/pkg/version"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(versionCmd)
+	versionCmd.Flags().BoolP("full", "f", false, "Returns the fully constructed version")
```

Comment:
Any idea on why a full version string would be needed? For the most part, I would think that interacting with the version command's formatted string would be sufficient, but I'm curious whether you had something else in mind.

Reply:
My main thought was that it could be used by some automation tools to confirm the current version in one line. That being said, I'm not too attached to keeping this.

Reply:
I don't have a strong opinion either - I was mainly interested in reducing scope if we don't have an immediate use case for it.